locales/es/homeGettingStarted.yml

```diff
@@ -0,0 +1,111 @@
+    Debe de haber una versión normal y una grande y una versión `primaria` para los botones importantes.
+    Así es como debe de verse una vez renderizado: (Esto es un ejemplo en vivo, haz click en ellos!)
+  second: |
+    Primero, hay que importar styled-components y crear un `styled.button`:
+
+    ```jsx
+    import styled from 'styled-components';
+
+    const Button = styled.button``;
+    ```
+
+    Esta variable `Button` aquí ahora es un componente de React que puedes usar como cualquier otro componente de React!
+    Este inusual sintaxis "backtick" es una característica nueva de JavaScript llamada una plantilla etiquetada literal (tagged template literal).
+    Sabes como ejecutar funciones con paréntesis? (`miFuncion()`) Bueno, ahora también puedes ejecutar funciones con "backticks"!
+    [Aquí hay una explicación](/docs/advanced#tagged-template-literals) de como funciona exactamente.
+
+    Si renderizas nuestro encantador componente ahora (Justo como cualquier otro componente: `<Button />`) esto es lo que obtienes:
+  third: |
+    Eso renderiza un botón! aunque no es muy bonito 😕 podemos hacerlo mejor,
+    démosle un poco de estilo y obtengamos la belleza escondida que lleva dentro!
+
+    ```jsx
+    const Button = styled.button`
+      border-radius: 3px;
+      padding: 0.25em 1em;
+      margin: 0 1em;
+      background: transparent;
+      color: palevioletred;
+      border: 2px solid palevioletred;
+    `;
+    ```
+  fourth: |
+    Como puedes ver, styled-components permite escribir CSS real en tu JavaScript. Esto significa que puedes usar todas
+    las características de CSS que usas y amas, incluyendo (pero de lejos no se limita a) media queries, todos
+    los pseudo-selectors, anidaciones, etc. También liga estilos a componentes, lo cual tiene propiedades interesantes.
+    (aprende más acerca [de las ideas detrás de styled-components aquí](/docs/basics#motivation))
+
+    El último paso es que tenemos que definir como se ve un botón primario.
+    Para hacer eso importamos `{ css }` de `styled-components` e interpolamos una función dentro de nuestra plantilla literal, la cual pasa los props de nuestro componente:
+
+    ```jsx
+    import styled, { css } from styled-components
+
+    const Button = styled.button`
+      border-radius: 3px;
+      padding: 0.25em 1em;
+      margin: 0 1em;
+      background: transparent;
+      color: palevioletred;
+      border: 2px solid palevioletred;
+
+      ${props => props.primary && css`
+        background: palevioletred;
+        color: white;
+      `}
+    `;
+    ```
+
+    Todo lo que estamos diciendo aquí es que cuando la pripiedad `primary` esté puesta queremos que añada un poco más de `css` a nuestro componente,
+    en este caso, cambiar el color de fondo y el color.
+
+    Eso es todo, hemos acabado! Echa un vistazo al componente terminado
+
+    ```react
```

Comment:
To put quick comment here, this is great to get the triple backticks without escaping, but

- the yaml wrapper is definitely not as nice, as I think it is with just having a tag in js
- this wouldn't allow us to use interpolations for JS 😢 

I'd personally say we should just make folders for each markdown with the appropriate files in there, translated and untouched apart from that

Reply:
Yeah. This part was quite troublesome. Interpolation was totally dead and I had to extract the components and just render them. It gave good results though

Reply:
Not sure how else can we get multiple line strings. Tried to use `json5` but failed and probably would have the same issues than `yaml`. Using just `json` was even worse, as all the lines had to be added in a single line with `\n`.